Adjust Pull Request Template (#1)
ivido/.github: .github/pull_request_template.md

_Delete this section up to and including the `------` text if your PR contains no breaking changes_
__BREAKING CHANGES__
This pull request contains the following breaking changes:
* Breaking change 1
* Breaking change 2 

------

> 

Reviewer: ```suggestion

```

- [ ] Does the pull request adhere to [code quality guidelines](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#764284756c264f1684bc7a9d49d054df)?
- [ ] Are [optimizations possible](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#bc27253ddf1b412fbdc7c2aca818bf17) for the content in this pull request?
- [ ] Is the content in the pull request [properly tested and validated](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#df8d441dae0044cb9414d2cf4a929a10)?
- [ ] Is the pull request [properly documented](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#131d60cb910040c9a482c84210478e8c), both [in the code](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#8a3533ffc8d14ecf91bb8c128ba35074) and in [documentation](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#5c6f8baa61cb4111981ef02d828da335)?

> - [ ] Do the changes in the codebase [break existing functionality or endpoints](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#dbc4674b18b84fdcb2692e22c16e69ce) that may be relevant to partners?

Reviewer: ```suggestion
- [ ] Do the changes in the codebase [break existing functionality or endpoints](https://www.notion.so/ivido/Pull-Request-Reviewing-b89fa374e2704319a0249d729db65eee#dbc4674b18b84fdcb2692e22c16e69ce)? Should any of our partners be informed about this change?
```

Author: I think we should make this a separate list item to hammer it down.

> - [ ] Do our partners need to be informed of this change?

Reviewer 2: ```suggestion
- [ ] Are our partners informed (if necessary)?
```
Iets in deze strekking

Author: Adjusted!

> __BREAKING CHANGE__ : This pull request contains a breaking change due to [reason]. You can remove this line if your pull request introduces no breaking changes.

Reviewer: ```suggestion
_Delete this section if your PR contains no breaking changes_
__BREAKING CHANGES__
This pull request contains the following breaking changes:
- Breaking change 1
- Breaking change 2 
```

------

> # TITLE OF PULL REQUEST GOES HERE

Reviewer: ```suggestion
```

A DESCRIPTION OF YOUR PULL REQUEST GOES HERE

* A bullet list of each adjustment, feature, and bug fix...
* goes here.
* Continued list.
* Add more as needed.

> 

Reviewer 2: Perhaps a short how-to-test message is in place?

Author: Good idea, will add it in!

# Pull Request Reviewing

To review thos pull request, please utilize the checklist below.

- [ ] Does the functionality described in the pull request work in practice?
- [ ] Does the functionality work in all cases?
- [ ] Can you break the implementation, ergo, could you find a corner case and break it?
- [ ] Does the contribution adhere to the agreed-upon format and package layout for the project?

> - [ ] Does the contribution adhere to the agreed-upon format and package layout for the project?

Reviewer: ```suggestion
```

> - [ ] Can the contribution be split-up into smaller methods, classes, or objects?

Reviewer: ```suggestion
- [ ] Should the contribution be split-up into smaller methods, classes, or objects?
```
